package.json

```diff
@@ -20,15 +20,15 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "@changesets/cli": "^2.26.0",
+    "@changesets/cli": "^2.26.1",
     "@svitejs/changesets-changelog-github-compact": "^1.1.0",
-    "@typescript-eslint/eslint-plugin": "^5.58.0",
-    "eslint": "^8.40.0",
-    "eslint-plugin-svelte": "^2.28.0",
+    "@typescript-eslint/eslint-plugin": "^5.60.0",
+    "eslint": "^8.43.0",
+    "eslint-plugin-svelte": "^2.31.0",
     "eslint-plugin-unicorn": "^47.0.0",
-    "playwright": "^1.34.3",
+    "playwright": "^1.35.1",
     "prettier": "^2.8.8",
-    "prettier-plugin-svelte": "^2.10.0"
+    "prettier-plugin-svelte": "^2.10.1"
   },
   "packageManager": "pnpm@8.6.0"
```

Comment:
Do we want to upgrade the version of pnpm as well?

Reply:
done